src/app/register/components/form-terms/form-terms.component.html

```diff
@@ -0,0 +1,34 @@
+<h3 class="mat-subheading-2" i18n="@@register.termsOfUse">Terms of Use</h3>
+
+<ng-container [formGroup]="form">
+  <mat-checkbox
+    class="orcid-checkbox-baseline"
+    color="primary"
+    formControlName="termsOfUse"
+    ><ng-container i18n="@@register.consentTo">I consent to the </ng-container
```

Comment:
@leomendoza123 the attribute preservedWitheSpaces won't work with ng-container you should use p instead

Reply:
good catch, the problem was that there was no space so I can still use ng-container so I don't add any extra HTML tag on the DOM.

Reply:
I have fixed this by adding spaces